src/components/ui-language-dialog.less

```diff
@@ -4,38 +4,21 @@
   margin: 0 0 25px 0;
 }
 
-.UiLanguageDialog-languageSwitch {
+.UiLanguageDialog-languages {
   display: grid;
   column-gap: 25px;
   grid-auto-flow: column;
 }
 
-.UiLanguageDialog-languageButton {
-  width: 100%;
-  height: auto;
-
-  &.ant-radio-button-wrapper {
-    border: none;
-  }
-
-  &.ant-radio-button-wrapper-checked {
-    background-color: @edu-selection-background-color;
-
-    &:focus,
-    &:focus-within {
-      box-shadow: none;
-    }
-
-    &:hover {
-      .m-box-shadow;
-    }
-  }
+.UiLanguageDialog-language {
+  padding: 10px 10px 5px 10px;
+  border-radius: @edu-border-radius-base;
```

Comment:
We might want to add `cursor: pointer;` here, as it is gone now that this is not a radio button anymore.

Reply:
True, added!